Cron/UpdateOrdersToProcessingV2.php

```diff
@@ -102,6 +102,11 @@ class UpdateOrdersToProcessingV2 {
      */
     protected $captureCommand;
 
+    /**
+     * @var \Razorpay\Magento\Model\Util\DebugMode
+     */
+    protected $debug;
```

Comment:
> 2.7. All non-public properties and methods SHOULD be private.

https://developer.adobe.com/commerce/php/coding-standards/technical-guidelines/
```suggestion
    private $debug;
```